DockDoor/Utilities/Window Management/WindowUtil.swift

```diff
@@ -248,6 +248,10 @@ enum WindowUtil {
 // MARK: - Cache Management
 
 extension WindowUtil {
+    static func getAllWindowsFromCache() -> [WindowInfo] {
```

Comment:
This is duplicating the getAllWindowsOfAllApps method, is there a reason this is a new method rather than being integrated into the existing methods?